template/greeter/.buildkite/pipeline.yml

```diff
@@ -1,11 +1,7 @@
-configs:
-  environments:
-    - &prod
-      agents:
-        queue: <%- prodBuildkiteQueueName %>
-      env:
-        ENVIRONMENT: prod
```

Comment:
We're okay losing the `ENVIRONMENT` here?

Reply:
Yeah, `greeter` is barebones so it doesn't control anything.